pages/trial.tsx

```diff
@@ -114,6 +114,9 @@ export default function Trial({ props }) {
         send_to: "send_to': 'AW-467110449/dWjCCL2x78ICELGU3t4B", // trialRequest conversion
```

Comment:
This addwords account should probably be moved to an environment variable AW-467110449/dWjCCL2x78ICELGU3t4B

Reply:
That sounds right.  I believe there is at least one elsewhere as well from setting up the paid search tracking.